lib/tyrant/railtie.rb

```diff
@@ -1,17 +1,18 @@
 module Tyrant
   class Railtie < Rails::Railtie
-      require "warden"
+    require "warden"
 
-      # DISCUSS: it will be configurable what user class etc. and might be moved to Ops.
-      # DISCUSS: this should actually use Tyrant::Session to log in and out user?
-      config.app_middleware.use Warden::Manager do |config|
-        Warden::Manager.serialize_into_session do |user|
-          user.id
-        end
+    # DISCUSS: this should actually use Tyrant::Session to log in and out user?
+    config.app_middleware.use Warden::Manager do |config|
```

Comment:
Another point is, doing this: `config.tyrant.user = ` force you to only have one model type for all users. With scopes, you can decide to use `User` for scope `:user` and `Client` for scope `:client` for example. Since Railtie is for Rails, we can keep a little magic there? (or add tyrant-rails gem :D)